tools/check-for-update.js

```diff
@@ -1,61 +1,63 @@
+const config = require('config');
+const exec = require('child_process').exec;
 const fetch = require('node-fetch');
 const fs = require('fs');
 const semver = require('semver');
 
-const exec = require('child_process').exec;
-
-const remote = 'mozilla-iot';
-
-const pkg = JSON.parse(fs.readFileSync('package.json', {encoding: 'utf-8'}));
+const pkg = require('../package.json');
 
-fetch(`https://api.github.com/repos/${remote}/gateway/releases`).then(res => {
-  return res.json();
-}).then(releases => {
-  // Assumes that releases are in chronological order, latest-first
-  return releases.filter(release => {
-    return !release.prerelease && !release.draft;
-  })[0];
-}).then(latestRelease => {
-  if (!latestRelease) {
-    console.error('No releases found');
-    return;
-  }
-  let releaseVer = latestRelease.tag_name;
-  let currentVer = pkg.version;
-  if (semver.lt(currentVer, releaseVer)) {
-    // do upgrade here woo
-    // download latestRelease.assets[:].browser_download_url
-    let gatewayUrl = null;
-    let nodeModulesUrl = null;
-    let validAssetRe = new RegExp(
-      `^https://github.com/${remote}/gateway/releases/download/` + releaseVer
-      + '/[a-z0-9_-]+.tar.gz$');
-    for (let asset of latestRelease.assets) {
-      if (!asset.browser_download_url.match(validAssetRe)) {
-        continue;
-      }
-      if (asset.name.match(/^gateway-[a-f0-9]+\.tar\.gz$/)) {
-        gatewayUrl = asset.browser_download_url;
-      }
-      if (asset.name.match(/^node_modules-[a-f0-9]+\.tar\.gz$/)) {
-        nodeModulesUrl = asset.browser_download_url;
-      }
+fetch(config.get('updateUrl'),
+      {headers: {'User-Agent': `mozilla-iot-gateway/${pkg.version}`}})
+  .then(res => {
```

Comment:
nit: should be consistent with line 13 (which means it wouldn't have the extra indentation either)